### ISSUE TITLE
Live usb?

### ISSUE THREAD

**Issue by shaswat8584** [Alex313031/ChromiumOS#7]
is it possible to make live usb of the chrome os fork?

**Comment by Alex313031** [comment]
@shaswat8584 Yes, write it to a usb, and then use gparted or similar to expand the STATEFUL partition to use the rest of the USB stick (normally only takes up 4GB, this it to keep the image size under 8GB so that it fits on an 8GB usb stick or larger)

**Comment by shaswat8584** [comment]
bro it works but i cant login it says something went wrong please help.

**Comment by Alex313031** [comment]
@Alex313031 as per the readme> https://github.com/Alex313031/ChromiumOS#readme, and the official Chromium documentation > https://blog.chromium.org/2021/01/limiting-private-api-availability-in.html you must add your google account to https://groups.google.com/u/1/a/chromium.org/g/google-browser-signin-testaccounts to login. I'll put a notice on future releases, because I can't expect everyone to read through the whole readme prior to using the OS.

**Comment by shaswat8584** [comment]
Yes bro when I enter my password it says "Something went wrong" help

**Comment by shaswat8584** [comment]
And I am using right password.

**Comment by Alex313031** [comment]
@shaswat8584 Did you add yourself to that group?
